tests/integrations/server/gc_worker.rs

```diff
@@ -71,6 +71,7 @@ fn test_physical_scan_lock() {
 fn test_applied_lock_collector() {
     let mut cluster = new_server_cluster(0, 3);
     cluster.run();
+    cluster.pd_client.disable_default_operator();
```

Comment:
What is this for?

Reply:
The test fails frequently when adds a new peer. The reason is PD's default operator already adds a new peer to the newly added store.

Reply:
Got it. Thanks!